public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -156,7 +156,7 @@ export class GeomapPanel extends Component<Props, State> {
     }
 
     if (options.controls !== oldOptions.controls) {
-      this.initControls(options.controls ?? { showZoom: true, showAttribution: true });
+      this.initControls(options.controls ?? { showZoom: true, showAttribution: true, scaleUnits: 'degrees' });
```

Comment:
Where is 'degrees' coming from?

Reply:
ah great catch! I was testing type checking and forgot to remove this after 🙈 Will fix now.